src/stdatm/state_parameters.py

```diff
@@ -118,7 +118,7 @@ def compute_kinematic_viscosity(
     temperature: Union[np.ndarray, Number], density: Union[np.ndarray, Number]
 ) -> Union[np.ndarray, Number]:
     kinematic_viscosity = (
-        (0.000017894 * (temperature / SEA_LEVEL_TEMPERATURE) ** (3 / 2))
+        (0.000017894 * (temperature / SEA_LEVEL_TEMPERATURE) ** 1.5)
```

Comment:
Does this make a significant difference on the benchmarks ?

Reply:
No it does not. But with no impact on readability, I consider there is no point in having a useless operation here.

Reply:
Ok ! I'll try to adopt the same approach